Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -350,13 +395,13 @@ at which point this block turns back to a PID controller but with tuned PID para
 <b>Note:</b> If an autotuning is ongoing, i.e., <code>inTunPro.y = true</code>,
```

Comment:
Should we also add the note that when there is setpoint change during the tuning process, the tuning will stop and the parameter will not change?

Reply:
Yes, this note is now handled by <code>assMes3</code> and <code>assMes4</code> in <code>Buildings.Controls.OBC.Utilities.PIDWithAutotuning.SystemIdentification.FirstOrderTimedelayed.ControlProcessModel</code>.
Note that when there is setpoint change during the tuning process, the tuning will still proceed but the parameter will not change.